pkgs/by-name/no/noto-fonts-cjk-serif/package.nix

```diff
@@ -3,23 +3,44 @@
 , fetchFromGitHub
 , nixosTests
 , gitUpdater
+
+, static ? false, # whether to build the static version of the font
 }:
 
 stdenvNoCC.mkDerivation rec {
   pname = "noto-fonts-cjk-serif";
   version = "2.003";
 
-  src = fetchFromGitHub {
-    owner = "notofonts";
-    repo = "noto-cjk";
-    rev = "Serif${version}";
-    hash = "sha256-E+Ic7XhomI6cUa+q77gQvMlaLvy+vgTq4NJ58/nPZtk=";
-    sparseCheckout = [ "Serif/Variable/OTC" ];
-  };
+  src =
+    let
+      sources = {
+        static = fetchFromGitHub {
+          owner = "notofonts";
+          repo = "noto-cjk";
+          rev = "Serif${version}";
+          hash = "sha256-mfbBSdJrUCZiUUmsmndtEW6H3z6KfBn+dEftBySf2j4=";
+          sparseCheckout = [ "Serif/OTC" ];
+        };
+        variable = fetchFromGitHub {
+          owner = "notofonts";
+          repo = "noto-cjk";
+          rev = "Serif${version}";
+          hash = "sha256-E+Ic7XhomI6cUa+q77gQvMlaLvy+vgTq4NJ58/nPZtk=";
+          sparseCheckout = [ "Serif/Variable/OTC" ];
+        };
+      };
```

Comment:
You can use one source (they are identical) with both directories in `sparseCheckout`.

Reply:
I know, but wouldn't that make users download more than what they need to, instead of doing the minimum in each case?

Reply:
Only the ones building from source, and not by much, then users would get just the right results from the cache.

Reply:
Indeed, I didn’t think about the cache. Thanks!